provider/app.py

```diff
@@ -157,6 +157,14 @@ def main():
     streamHandler.setFormatter(formatter)
     logger.addHandler(streamHandler)
 
+    # also log to file if /logs is present
+    if os.path.isdir('/logs'):
+        fh = logging.FileHandler('/logs/kafkatriggers.log')
```

Comment:
Does this overwrite old logs? Maybe a timestamp in the filename would be useful to preserve old logs